gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-agp = "8.3.0"
+agp = "8.2.2"
```

Comment:
Is is a merge error or it's actually something you need ?

Reply:
I have this error : The project is using an incompatible version (AGP 8.3.0) of the Android Gradle plugin. Latest supported version is AGP 8.2.2
So I was forced to change the agp from 8.3.0 to 8.2.2.
But I think that the problem comes from my android studio version so you can leave agp =8.3.0